scripts/build-ios.sh

```diff
@@ -92,7 +92,6 @@ pushd build-ios
 # Configure CMake project
 cmake "$PROJECT_ROOT" -GXcode \
     -DCMAKE_SYSTEM_NAME=iOS \
-    -DCMAKE_TOOLCHAIN_FILE="$PROJECT_ROOT/src/realm-core/tools/cmake/ios.toolchain.cmake" \
```

Comment:
We need to use https://github.com/realm/realm-core/blob/master/tools/cmake/xcode.toolchain.cmake

Reply:
Fixed in https://github.com/realm/realm-dart/pull/303/commits/ad80433524169c25772b9f4b0432cdad316297e3